src/book/02-system/02-subsystems/06-behaviour.mdx

```diff
@@ -80,3 +80,13 @@ A script consists of frames (also called targets) which specify the position of
 ScriptTuner also allows for "locking" and "unlocking" servos on the robot. When a servo is "locked", its position can only be changed in ScriptTuner. When it is "unlocked", the servo can be moved physically on the robot, which will update the position value in the script. Locking a servo involves emitting a ServoCommand for the current position of the servo. Unlocking the servo involves emitting a ServoCommand with zero torque and gain.
 
 To learn how to use ScriptTuner, read the [ScriptTuner guide](/guides/main/tuning-and-running-scripts#script-tuner).
+
+## Path planning
+
+[SimpleWalkPathPlanner](https://github.com/NUbots/NUbots/tree/main/module/behaviour/planning/SimpleWalkPathPlanner) is the current implemenation of path-planning for the robot and currently (as of **_09/2021_**) uses a combination of messages from vision and localisation in order to determine messages to emit which motion can then interpret to control the positions the robot moves to on the field during play.
```

Comment:
```suggestion
We use a [SimpleWalkPathPlanner](https://github.com/NUbots/NUbots/tree/main/module/behaviour/planning/SimpleWalkPathPlanner) to implement path-planning for the robot. The path planner uses a combination of messages from Vision and Localisation to determine which messages to emit. The motion subsystem can then interpret these messages to control the positions and locations the robot moves to on the field during play.
```
I'm not sure we need to specify a date here... It will probably get out of date at some point although including dates isn't generally the done thing

Reply:
Yeah not sure on the date thing, probs best to remove but I like the concept. I was thinking Git already dates things so we know when it was updated. But it might be cool to have some way of seeing on the site itself when things have last been updated, maybe also who contributed so people know who to ask for more info. Maybe some sort of footer on the page.

Reply:
I'd also recommend linking to the vision, localisation and motion pages where you mention the systems (if they haven't already been linked on the page).

Reply:
That's an interesting idea @ysims are you thinking like a gitblame? We could look into that...

Reply:
I've a similar idea of listing who contributed to a page since the beginning, primarily to encourage people to contribute by publicly giving credit. But knowing who to contact about a page is also a good reason to have that. And a last update date on each page would be good too 👍.

Reply:
I've put it on the todo list, we can look into getting it done :D
https://github.com/NUbots/NUbook/issues/186